python/jsbeautifier/javascript/beautifier.py

```diff
@@ -670,8 +691,8 @@ def handle_start_block(self, current_token):
     check_token.type != TOKEN.EOF and not (
          check_token.type == TOKEN.END_BLOCK and check_token.opened == current_token))
 
-        if (self._options.brace_style == 'expand' or (self._options.brace_style ==
-                                                  'none' and current_token.newlines)) and not self._flags.inline_frame:
+        if (self._options.brace_style == 'expand' or self._options.brace_style == 'expand-all' or
+                (self._options.brace_style == 'none' and current_token.newlines)) and not self._flags.inline_frame:
```

Comment:
New feature to have same behavior as that of brace style option with value expand.